source/plugin/scheduler.rst

```diff
@@ -163,3 +165,139 @@ In addition, there are a few other operations that are safe to do asynchronously
 
 * Independent network requests
 * Filesystem I/O (excluding files used by Sponge)
+
+Compatibility with other libraries
+==================================
+
+.. _ExecutorService: https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html
+
+As your plugin grows in size and scope you might want to start using one of the many concurrency libraries available 
+for Java and the JVM.
+These libraries do tend to support Java's ExecutorService_ as a means of directing on which thread the task is executed.
+
+For compatibility Sponge can create an ``ExecutorService`` using the ``Scheduler#createSyncExecutor(Object plugin)`` or 
+``Scheduler#createAsyncExecutor(Object plugin)`` methods which executes tasks on Sponge's synchronous or asynchronous 
```

Comment:
Bit of a run-on sentence.
For compatibility, Sponge can create an \`\`ExecutorService\`\` using the ... or ... methods, which executes tasks on Sponge's ...